.github/workflows/deploy-infra.yaml

```diff
@@ -106,7 +106,7 @@ jobs:
     strategy:
       max-parallel: 1
       matrix:
-        environment-type: ${{ fromJson(needs.setup.outputs.matrix) }} ["staging", "production"]
+        environment-type: ${{ fromJson(needs.setup.outputs.matrix) }}
```

Comment:
This should not show up if you merge in master I think, it was fixed on thursday

Reply:
On the other hand, I think this shows up because I merge in `master`. The merge must have ben a rebase or something.

Reply:
I don't think it's an issue right? As long as the code is the same,